content/docs/command-reference/remote/modify.md

```diff
@@ -122,12 +135,6 @@ these settings, you could use the following options:
   $ dvc remote modify myremote secret_access_key my-secret_access_key
   ```
 
-- `url` - remote location URL:
```

Comment:
Let's not optimize out this? People don't read texts as you expect. A lot of them would just jump to the section they need.